.github/workflows/create-rc-tag.yml

```
@@ -0,0 +1,101 @@
+name: Create Rc Tag
+# Required secrets for this workflow: MAINTAIN_USERNAME, MAINTAIN_EMAIL, SLACK_TOKEN, MAINTAIN_TOKEN, and a var: SLACK_TAG_CHANNELS.
+
+env:
+  CHANGELOG_FILE: 'temp-changelog.txt'
+
+on:
+  workflow_dispatch:
+    inputs:
+      channel:
+        type: choice
+        description: 'The channel to get the version to'
+        required: true
+        default: 'cloud'
+        options:
+          - 'ga'
+          - 'cloud'
+          - 'beta'
+jobs:
+  create-tag:
+    runs-on: ubuntu-20.04
```

Lets use the latest we can
```suggestion
    runs-on: ubuntu-22.04
```

---

Done